administrator/language/en-GB/plg_editors_tinymce.ini

```
@@ -74,6 +74,6 @@ PLG_TINY_XML_DESCRIPTION="TinyMCE is a platform independent web based JavaScript
 ; External Plugins
 PLG_TINY_FIELD_LABEL_EXTERNAL="External Plugins"
 PLG_TINY_FIELD_EXTERNALPLUGINS_LABEL="External Plugin URLs"
-PLG_TINY_FIELD_EXTERNALPLUGINS_DESC = "Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or a complete absolute URL."
+PLG_TINY_FIELD_EXTERNALPLUGINS_DESC="Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or a complete absolute URL."
```

```suggestion
PLG_TINY_FIELD_EXTERNALPLUGINS_DESC="Provide a pair made up of a Plugin Name and source path to the Plugin's JavaScript file. The Path should be relative to the site's root, or an absolute URL."
```

by definition an absolute url is complete. so you dont need that word

---

thanks @brianteeman